pytorch/inference/docker/1.10/py3/Dockerfile.e3.cpu

```diff
@@ -13,6 +13,7 @@ ARG PT_TORCHVISION_URL=https://framework-binaries.s3.us-west-2.amazonaws.com/pyt
 ENV LANG C.UTF-8
 ENV LD_LIBRARY_PATH /opt/conda/lib/:$LD_LIBRARY_PATH
 ENV PATH /opt/conda/bin:$PATH
+ENV SAGEMAKER_SERVING_MODULE sagemaker_pytorch_serving_container.serving:main
```

Comment:
This needs to be added into the SageMaker serving dockerfile, not the E3 dockerfile.